src/server/services/about/views/service.test.js

```diff
@@ -0,0 +1,214 @@
+import { nunjucksTestEnv } from '~/test-helpers/component-helpers.js'
+// eslint-disable-next-line n/no-unpublished-import
+import { toMatchFile } from 'jest-file-snapshot'
+
+expect.extend({ toMatchFile })
+
+describe('Page rendering test for services page', () => {
+  function passThrough(item) {
+    return item
+  }
+
+  it('renders the page for a frontend service', () => {
+    const rendered = nunjucksTestEnv.render(
+      'services/about/views/service.njk',
+      {
+        getAssetPath: passThrough,
```

Comment:
Not sure if it would make this a little more awkward or better but is there not work in `__fixtures__` that could be used here?

Reply:
cool, will take a look

Reply:
They're all tied to certain services. not quite the flexibility. something we could look at going forward.